menyer/for_goodness_shake/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>For Goodness Shake Meny</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <main>
+        <h1>Meny</h1>
+        <section class="container menu">
+            <div class="menuSection milkshakes">
+                <h2>Milkshakes</h2>
+                <p class="menuItems">
+                    Vanilj <br>
+                    Choklad <br>
+                    Päron <br>
+                    Jordgubb <br>
+                    Hallon <br>
+                    Banan <br>
+                    Citron & Lakrits <br>
+                </p>
+                <p class="itemPrices milkshakePriser">
+                    35 Kr <br>
+                    35 Kr <br>
+                    35 Kr <br>
+                    35 Kr <br>
+                    35 Kr <br>
+                    35 Kr <br>
+                    35 Kr <br>
+                </p>
+                
+            </div>
+
+            <div class="menuSection hamburgare">
+                <h2>Hamburgare</h2>
+                <p class="menuItems">
+                    Hamburgare <br>
+                    Ostburgare <br>
+                    Baconburgare <br>
+                    Extra allt <br>
+                </p>
+                <p class="itemPrices hamburgarePriser">
+                    74 kr <br>
+                    79 kr <br>
+                    79 kr <br>
+                    109 kr <br>
+                </p>
+            </div>
+
+            <div class="menuSection tillbehor">
+                <h2>Tillbehör</h2>
+                <p class="menuItems">
+                    Extra pommes <br>
+                    Extra kött <br>
+                    Sallad <br>
+                </p>
+                <p class="itemPrices tillbehorPriser">
+                    15 kr <br>
+                    35 kr <br>
+                    25 kr <br>
+                </p>
```

Comment:
Works but a bit unconventional. Normally a `table` could be used here.

Reply:
I've removed what I had done and put it in all in a table instead.